x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality_details/details/fields_list.tsx

```diff
@@ -32,7 +32,12 @@ export function FieldsList({
       <EuiFlexGroup direction="column" gutterSize="none">
         {fields.map(({ fieldTitle, fieldValue, isLoading: isFieldLoading, actionsMenu }, index) => (
           <Fragment key={index + fieldTitle}>
-            <EuiFlexGroup>
+            <EuiFlexGroup
+              data-test-subj={`datasetQualityDetailsFieldsList-${fieldTitle
```

Comment:
Will this be used in FullStory? Or how do we envision to use this?

Reply:
For tests, like to check if the page is Integration or not
I check for presence of 2 rows, Integration Name and Version